cogs5e/models/initiative/combatant.py

```diff
@@ -466,12 +466,17 @@ def __init__(self,
         self._monster_id = monster_id
 
     @classmethod
-    def from_monster(cls, monster, ctx, combat, name, controller_id, init, private, hp=None, ac=None):
+    def from_monster(cls, monster, ctx, combat, name, controller_id, init, private, cust={}, stats=None, skills=None, saves=None, hp=None, ac=None):
```

Comment:
This should probably be `cust=None` and check `if cust:` later on.

https://florimond.dev/en/posts/2018/08/python-mutable-defaults-are-the-source-of-all-evil/

Reply:
Actually I just realized that cust isn't used there anymore, so it can be gotten rid of altogether. Sorry I didn't catch that sooner.